bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -676,5 +692,157 @@ contract("Bridge NFT", async function(accounts) {
         );
       });
     });
+
+    describe("claim", async function () {
+      let tokenAddress;
+      let symbol;
+      let name;
+      let baseURI;
+      let contractURI;
+      const tokenId = 1;
+      const blockHash = utils.getRandomHash();
+      const transactionHash = utils.getRandomHash();
+      const logIndex = 1;
+      let sideTokenAddress;
+
+      beforeEach(async function () {
+        symbol = await this.token.symbol();
+        name = await this.token.name();
+        baseURI = await this.token.baseURI();
+        contractURI = await this.token.contractURI();
+        tokenAddress = this.token.address;
+
+        // Side NFT token contract is created.
+        let receipt = await this.sideBridgeNft.createSideNFTToken(
+            tokenAddress, symbol, name, baseURI, contractURI, {from: bridgeManager}
+        );
+        utils.checkRcpt(receipt);
+        sideTokenAddress = receipt.logs[0].args[0];

```

Comment:
Maybe its easier to use truffle-assertions and if it's the right one set sideTokenAddress? https://www.npmjs.com/package/truffle-assertions#exported-functions
I had to refactor many tests because i added another event and i was using the index just like this.
We may use other librarys if you prefer, or we can just stick with the indexes

Reply:
yeah I used this bit to code it faster, but I agree in that it's pretty unmantainable (if the event changes, this breaks) - so I'll just leave it with the truffleAssert like I did before